conceptnet5/nodes.py

```diff
@@ -130,7 +130,7 @@ def get_uri_language(uri):
     """
     if uri.startswith('/a/'):
         return get_uri_language(parse_possible_compound_uri('a', uri)[0])
-    elif uri.startswith('/c/'):
+    elif uri.startswith('/c/') or uri.startswith('/x/'):
```

Comment:
`test_vectors` have a couple of places where we check if terms start with '/c'. Should these be changed as well?

Reply:
Can you remind me what `test_vectors` is?

Reply:
Oh sorry, I meant `conceptnet5/tests/small-build/test_vectors.py`.